client/src/atoms/BaseButton.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <button class="btn">{{ text }}</button>
+</template>
+
+<script>
+export default {
+  name: 'BaseButton',
+  props: {
+    text: {
+      type: String,
+      required: true,
+    },
+  },
+};
+</script>
+
+<style scoped>
+.btn {
+  background-color: #007bff;
+  color: white;
+  padding: 10px;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
```

Comment:
This style shouldn't be necessary with an ionic button.

Reply:
I have removed the custom css.